app/client/keybase/keystore.go

```diff
@@ -283,40 +284,47 @@ func (keybase *badgerKeybase) StoreChildFromSeed(seed []byte, childIndex uint32,
 
 			return tx.Set(addrKey, keypairBz)
 		})
-		return err
-	}
 
-	// Re-encrypt child key with passphrase and hint
-	return keybase.db.Update(func(tx *badger.Txn) error {
-		// Get the private key hex string from the child key
-		privKeyHex, err := childKey.ExportString("") // No passphrase by default
-		if err != nil {
-			return err
-		}
+		return childKey, err
+	} else {
```

Comment:
Why did you reintroduce the else statement?

Since we return in the `if`, its easier to read without it.